packages/cowsay.rb

```diff
@@ -0,0 +1,13 @@
+require 'package'
+
+class Cowsay < Package
+  description 'cowsay is a program that generates ASCII pictures of a cow with a message.'
+  homepage 'https://en.wikipedia.org/wiki/Cowsay'
```

Comment:
This is not really a home page, per say, but it does adequately describe cowsay.  Probably should use https://github.com/tnalpgge/rank-amateur-cowsay/ instead. :)